custom_dc/data_cloud.md

```diff
@@ -117,23 +118,25 @@ If you are prompted to install the Cloud Resource Manager API, press `y` to acce
 
 If you have not made changes that require a local build, and just want to run the pre-downloaded image, from your repository root, run:
 
-```shell
+<pre>
 docker run -it \
---env-file $PWD/custom_dc/cloudsql_env.list \
+--env-file $PWD/custom_dc/env.list \
+-v <var>OUTPUT_DIRECTORY</var>:/<var>OUTPUT_DIRECTORY</var> \
```

Comment:
For GCS paths, there is nothing to mount. So you can remove this.

Reply:
Done.